src/commands/guess.ts

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable max-len */
+// pls suggestions, i want make some kind of json file containing all the replies (it would make the translation easier)
+import dayjs from 'dayjs';
+import { CommandInteraction, MessageActionRow, MessageButton } from 'discord.js';
+import { RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-types/v10';
+import { Command } from '../interfaces/Command';
+import { checkOrRegisterUser, getDay, getWord, played } from '../database';
+import { runAtMidnight } from '../utils/runner';
+import { awaitMessage } from '../utils/msgCollector';
+import { isValid } from '../utils/checkWord';
+import { toDefault, toEmoji } from '../utils/converters';
+import { plataform } from '../utils/plataform';
+import { hasPermissions } from '../utils/permissions';
+import { square } from '../utils/emotes.json';
+
+let usersTries: Record<string, Record<string, string | string[]>> = {};
```

Comment:
I think this should be typed to something specific like: `Record<string, { id: string, attempts: string[] }>`
If you do this, remember to type the `playingUser` const.

Reply:
In this case, `playingUser`'s type would be `{ id: string, attempts: string[] }`, right?

Reply:
Yeah